docs/features/configurationReload.md

```diff
@@ -0,0 +1,21 @@
+## Triggering Configuration Reload
+
+You have the following option to trigger a configuration reload:
+
+- via the user interface: `Manage Jenkins -> Configuration -> Reload existing configuration`
```

Comment:
In the option “via the user interface”, it is “Configuration as Code” instead of “Configuration" (on Jenkins 2.479.1).